docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+  token: "1234abcd5678efgh9"
+  ca_path: ""
+  ca_pins:
+  - sha256:1234abcd5678efgh910ijklmnop
+  join_method: token
+storage:
+  directory:
+    path: /var/lib/teleport/bot
+    symlinks: secure
+    acls: try
+destinations:
+  - directory:
+      path: /opt/machine-id
+    configs:
+      - ssh_host_cert:
+          principals: [nodename.my.domain.com]
+debug: false
+auth_server: example.teleport.sh:443
+certificate_ttl: 1h0m0s
+renewal_interval: 20m0s
+oneshot: false
+```
+
+Note the `renewal_interval` field, which will determine the cadence in which new certificates will be generated. Machine ID uses a default of 20 minutes for the `renewal_interval`, however this field may be adjusted as needed.
+
+## Step 4/4 Start Machine ID and create a host certificate
+
+Using your configuration file, The `tbot start -c` command will start running Machine ID in a loop,
+writing renewable certificates to /var/lib/teleport/bot according to your configuration, and the short-lived certificates including the Host Certificate your bot will use to `/opt/machine-id`.
+
+In a production environment you will want to run Machine ID in the background using a service manager like systemd. However, in this guide you will run it in the foreground to better understand how it works.
+
+```code
+$ tbot start -c tbot.yaml
+```
+
+Now that Machine ID has successfully started, let's investigate the /opt/machine-id directory to see what was written to disk.
+
+```code
+$ ls -1 /opt/machine-id
+identity
+key
+key-cert.pub
+key.pub
+known_hosts
+ssh_config
+ssh_host
+ssh_host-cert.pub
+teleport-database-ca.crt
+teleport-host-ca.crt
+teleport-user-ca.crt
+tlscert
+```
+
+The public host certificate and private key Machine ID will be generating can be found within the `ssh_host-cert.pub` and `ssh_host` file. To confirm this,
+view the public certificates contents by entering the following command:
+
+```code
+$ sudo ssh-keygen -L -f /opt/machine-id/key-cert.pub
+```
+
+The contents of the public certificate will appear, and will contain a `principals` field which will contain the `nodename.my.domain.com` domain entered in a previous step.
+Your output will resemble the following:
+
+```
+        Type: ssh-rsa-cert-v01@openssh.com host certificate
+        Public key: RSA-CERT SHA256:abcde123efghijk567lmnop
+        Signing CA: RSA SHA256:abcde123efghijk567lmnop (using rsa-sha2-512)
+        Key ID: ""
+        Serial: 0
+        Valid: from 2022-09-30T13:16:36 to 2022-09-30T17:17:36
+        Principals:
+                nodename.my.domain.com
+        Critical Options: (none)
+        Extensions:
+                x-teleport-authority UNKNOWN OPTION (len 55)
+                x-teleport-role UNKNOWN OPTION (len 8)
+```
+
+## Next Steps
+
+Now that Host Certificates have been created with machine ID, they can be applied to OpenSSH configurations as needed. For more information regarding the next steps
+to connect to OpenSSH with Machine ID, see the following documentation:
```

Comment:
Maybe a slight reword since these are mostly not Machine ID specific?

```suggestion
to connect to OpenSSH with Teleport, see the following documentation:
```